sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ImmutableStorageWithVersioningTests.java

```diff
@@ -145,7 +149,7 @@ public void setup() {
         vlwBlob.upload(new ByteArrayInputStream(new byte[0]), 0);
     }
 
-    public static final class Body {
+    public static final class Body implements JsonSerializable<Body> {
```

Comment:
Probably should have shared types for both the async and sync test, this looks to be duplicated right now.